server/routes/admin/users/userInfo.js

```diff
@@ -9,21 +9,7 @@ router.get("/:id", (req, res) => {
       "address.user_id",
       "users.id"
     )
-    .select(
-      "users.id",
-      "users.username",
-      "users.email",
-      "users.first_name",
-      "users.last_name",
-      "users.is_admin",
-      "users.avatar_url",
-      "address.phone",
-      "address.street",
-      "address.city",
-      "address.state",
-      "address.pin_code",
-      "address.country",
```

Comment:
we don't need the address details?

Reply:
This all get passed along when i select the entire values. So the code gets small too and values are passed safely too